test_launch_ros/test/test_launch_ros/frontend/test_node_frontend.py

```diff
@@ -1,3 +1,4 @@
+# Copyright 2020 Rover Robotics c/o Dan Rose
```

Comment:
@tfoote do we accept authors in the copyright notice? (besides of including the copyright holder in it)

Reply:
I don't @tfoote wants to weigh in: ["There are many articles covering this topic online from lawyers and other experts"](
https://answers.ros.org/question/349677/when-should-i-add-a-copyright-at-the-top-of-a-source-file/)

I think with the Apache 2.0 license this might actually belong in a NOTICE file instead of the license header.

Reply:
We don't usually add authors to do copyright note.
I think that we do that to avoid having a long list of people in every source file.
That's different to the list of copyright holders, because that last one has legal implications.

I cannot merge a PR that doesn't follow the usual style of the copyright note without asking to the team if we accept this new style ...
I don't understand what are you trying to point out with [this](https://github.com/ros2/launch_ros/pull/146#discussion_r428882724) comment ...

Reply:
> We don't usually add authors to do copyright note.
> I think that we do that to avoid having a long list of people in every source file.
> That's different to the list of copyright holders, because that last one has legal implications.

Okay. How would you propose I proceed?

I would omit the copyright except for I don't want to falsely imply that OSRF is the copyright holder.

>  I don't understand what are you trying to point out with [this](https://github.com/ros2/launch_ros/pull/146#discussion_r428882724) comment ...

I'm pointing out there's no mandated style and explicitly no ROS community standard. By all modern accounts, a copyright notice is not even needed so it's just a courtesy allowing the author to be contacted.

Reply:
Though the "c/o" suffix seems odd, I don't see how it could cause an issue. The linters are happy. We can certainly have individuals (instead of organizations) as copyright holders as well.

Reply:
I think I didn't explain what I meant correctly, sorry about that.

> I would omit the copyright except for I don't want to falsely imply that OSRF is the copyright owner. They're just licensed to use this code under the Apache 2 license.

Updating the copyright notice is correct.

> a copyright notice is not even needed so it's just a courtesy allowing the author to be contacted.

I don't think that's the rationale why we have a copyright notice in all source files (IIUC it's not intended to be a point of contact to Authors).

> We can certainly have individuals (instead of organizations) as copyright holders as well

We can have individuals as copyright holders, but AFAIU adding an individual after "c/o" suffix doesn't mean he's a copyright owner. What I understand, is that he did the contribution on behalf of the copyright owner previously mentioned. @rotu please clarify if that's the case.

> Though the "c/o" suffix seems odd, I don't see how it could cause an issue. The linters are happy

The "c/o" suffix is definitely what I find odd.
If this is meant to retribute contributors for their work, I don't think that the copyright notice is the right place to do it (IMO, it adds confusion on who is the copyright holder).

----

If somebody else approves and merges, I won't block on this.

Reply:
I have to agree that it is odd.  While we don't have a formal declaration of what this should be, we do have the common practice of what all of our other files do.  So I'd say unless there is a good reason to be different, this should follow those common standards.

Just to be totally clear, I'd change the line to be:

```
# Copyright 2020 Rover Robotics
```

Reply:
I'll change it to the legal entity that owns the copyright